### DISCUSSION TITLE
Can't import module to React

### DISCUSSION THREAD

**Discussion by FeduniakVitalii** [mdbootstrap/TW-Elements-React#160] [Support from community]
I'm using React with hundreds import, but this library get error - The current file is a CommonJS module whose imports will produce 'require' calls; however, the referenced file is an ECMAScript module and cannot be imported with 'require'. Consider writing a dynamic 'import("tw-elements-react")' call instead.
  To convert this file to an ECMAScript module, add the field `"type": "module"` to 'c:/Repositories/PSFRN/TMS-React/package.json'.ts(1479)

**Comment by Trochonovitz** [comment]
Have you tried add field `"type: "module"` to your `package.json` file?
